roles/v2/mining-proxy/src/main.rs

```diff
@@ -144,6 +151,67 @@ pub fn initialize_r_logic() -> RLogic {
     }
 }
 
+mod args {
```

Comment:
Can you put the `mod args` into a separate file: `roles/v2/mining-proxy/src/args.rs`?